tests/conftest.py

```diff
@@ -0,0 +1,6 @@
+from pytest import fixture
+
+
+@fixture
+def api_key() -> str:
+    return "biteme-api-key"
```

Comment:
was wondering if this worked, but you're testing it on a free Bite so that's why it works :)

Reply:
Yep! I would love it if you wanted to set a default API key for freebie bites instead of me using something that I hope isn't taken. Or you could omit the API key in the path on your end or take it as a query parameter instead of a path parameter.

Reply:
Checking our API it doesn't do the API key check against the user's profile if it's a free Bite so a random string like  this is fine.